### ISSUE TITLE
Backend Developer/Data Science/ ML looking for a team

### ISSUE THREAD

**Issue by Dhiraj240** [fix-the-internet/spring-mvp-lab#74]
** Name **
Dhiraj Sharma

** Skillset **
Data Science, Machine Learning, Python, Java
** Short Bio **
I have been contributing to opensource since last 2 years and was selected as a Student Developer in Summer of Code in Space program organised by the European Space Agency. I was among 17 selected developers where I worked on Java, Networking and motion compensation algorithm deployment for Solar flares prediction. I had also used docker containers and sphinx documentation to write the technical aspects of the project.

I am Core Developer in coala where I used python programming for linting tools, used antlr and worked on lexer, parser combination for python-based libraries. Moreover, I reviewed more than 25+ PRs.
I was appointed as Google Code In 2018 mentor in this organisation.

I am also the part of the Public Lab organisation's mentor and review team where worked on frontend and backend.
Here I was appointed as Google Summer of Code 2019 mentor.

I have done internships in Wipro on Machine Learning test Automation using NLP, Bharat Electronics Limited in JDBC, SQL injection.

I am comfortable in working with new technologies in order to enhance my skillset.
As of now, I am placed as a Data Engineer in a good firm during campus placement.
In this last semester of my final year, I am looking for such great opportunities where I can devote my precious time in staying on the track. 
Moreover, I have no setbacks and will finish the required work on time once I get selected.
** Why Work With Me **

** Location **
India UTC+5:30
** University / Work Affiliation **
Incoming MS in James Cook
** Github **
https://dhiraj240.github.io/
https://github.com/dhiraj240

I have not applied to the program since I am was looking for a team for long and someone from my group has betrayed so please look at your requirements.
As there is no loss of yours. Moreover, it will help me for my future and yours too.
Please, a humble request.

**Comment by mraza99** [comment]
Hi! We are looking for someone with web development/app development experience to help us with our project.  #82 

Would you be interested?

**Comment by Dhiraj240** [comment]
Yes, please send me the requirements on dhiraj.8.sharma@gmail.com. You can contact me on slack too id is Dhiraj

**Comment by griffiecat** [comment]
Hi Dhiraj are you on a team yet? I am looking for someone with technical skills!

**Comment by Dhiraj240** [comment]
@griffiecat Yes, today only I am going to pair up.

**Comment by salbadillo** [comment]
Hi Dhiraj,

Are you on a team yet?
